.github/workflows/pytest.yml

```
@@ -0,0 +1,25 @@
+name: Pytest PR
+on:
+  pull-request:
+    branches:
+      - main
+jobs:
+  run-test-code:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          cache: 'pip'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Test with pytest
```

How about using the github action syntax?

```suggestion
      - name: Test with pytest
         env:
            KLAYTN_ETL_RUN_SLOW_TESTS:  True
```

---

https://docs.github.com/en/actions/learn-github-actions/environment-variables